lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -306,15 +310,16 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
   defp unavailable_secret?(_socket, ""), do: false
 
   defp unavailable_secret?(socket, preselect_name) do
-    not session?(socket, preselect_name) and
-      not app?(socket, preselect_name) and
-      not hub?(socket, preselect_name)
+    get_secret(socket, preselect_name) != []
```

Comment:
This will always return true. Which means we are always prefilling the secret name.